docs/edgeql/ddl/migrations.rst

```diff
@@ -7,60 +7,103 @@ Migrations
 This section describes the DDL commands pertaining to migrations.
 
 
-CREATE MIGRATION
-================
+START MIGRATION
+===============
 
 :eql-statement:
 
-Create a new migration.
+Start a migration block.
 
 .. eql:synopsis::
 
-    [ WITH [ <module-alias> := ] MODULE <module-name> ]
-    CREATE MIGRATION <name> TO "{"
+    START MIGRATION <name> TO "{"
         <sdl-declaration> ;
         [ ... ]
     "}" ;
 
+Parameters
+----------
+
+:eql:synopsis:`<sdl-declaration>`
+    Module contents defined using the declarative :ref:`EdgeDB schema
```

Comment:
It's not necessarily contents of one module. Can we say "Schema defined using ..."?